internal/distro/fedora32/distro.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 const defaultName = "fedora-32"
+const releaseVersion = "32"
```

Comment:
I think we should reference `releaseVersion` in the other constants (`defaultName`, `modulePlatformID`, `ostreeRef`). Something for a follow-up.